content/correspondence/getting-started/developer-guides/serviceowner/_index.en.md

```
@@ -0,0 +1,77 @@
+
+**TIP**: Verify your configurations using the [Postman collection](https://github.com/Altinn/altinn-correspondence/blob/main/altinn-correspondence-postman-collection.json), substituting the test tokens with your own Altinn tokens (See "Login to Maskinporten (Initialize)" request in Authenticator folder).
+
+### 3. Altinn API key and access to scopes {#get-an-altinn-api-key}
+In order to use Altinn Correspondence you need a subscription key. Technically, this is an API key that must be included in the header of the request `Ocp-Apim-Subscription-Key`, to verify that you have the right to use the Correspondence API. Without this key, your request will be denied.
+To authenticate and ensure that you can perform operations via the Correspondence API, Altinn must grant you access to the necessary scopes. This ensures that only authorized clients can send and receive files, thereby maintaining the security of the service. The following scopes are used to send and/or receive messages:
+- `altinn:correspondence.write` 
+- `altinn:correspondence.read` 
+
+To obtain an Altinn API key and access to scopes, you must submit a request to: [servicedesk@altinn.no](mailto:servicedesk@altinn.no).
+The request must include the scopes you need. Note that you may require more than just read/write access for Correspondence. A complete list of scopes can be found here:
+https://docs.altinn.studio/api/authentication/digdirscopes/
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)